.github/actions/RunAutomation/UpdateAppBaselines/run.ps1

```diff
@@ -0,0 +1,19 @@
+[Diagnostics.CodeAnalysis.SuppressMessageAttribute('PSReviewUnusedParameter', 'runParameters', Justification = 'The parameter is always passed to the script')]
+param (
+    [Parameter(Mandatory=$true)]
+    $runParameters
+)
+
+# BC Container Helper is needed to fetch the latest artifact version
+Install-Module -Name BcContainerHelper -AllowPrerelease -Force
+Import-Module BcContainerHelper -DisableNameChecking
+Import-Module $PSScriptRoot\..\..\..\..\build\scripts\EnlistmentHelperFunctions.psm1
```

Comment:
This should be better. Otherwise, refactoring scripts becomes a nightmare.

Reply:
That was my thinking as well.

Reply:
It's actually a different issue. It's not about initializing some repo paths, it's about accessing build script from actions in the same repo.

Reply:
No better solution for now. We'll re-iterate on a better structure later on.